src/Filtering/itktubeConvertShrunkenSeedImageToListFilter.h

```diff
@@ -105,7 +105,7 @@ class ConvertShrunkenSeedImageToListFilter : public ProcessObject
   void
   PrintSelf(std::ostream & os, itk::Indent indent) const override;
 
-  void
+  virtual void
```

Comment:
Was this made virtual to mirror [setup from ProcessObject](https://github.com/InsightSoftwareConsortium/ITK/blob/122bd358611b48e04ca88bb40378453c3264510a/Modules/Core/Common/include/itkProcessObject.h#L784-L785)? Or for some other reason?